src/amazon-bedrock-agentcore-mcp-server/README.md

```diff
@@ -0,0 +1,410 @@
+# AWS Labs Amazon Bedrock Agent Core MCP Server
+
+An AWS Labs Model Context Protocol (MCP) server for Amazon Bedrock Agent Core. This MCP server allows you to build, deploy, and manage intelligent agents with advanced capabilities like memory, OAuth authentication, and gateway integrations.
+
+### Features:
+
+Transform and modernize your existing agent code from any framework (Strands, Langgraph, LangChain, LlamaIndex, AutoGen, CrewAI, etc.) into production-ready AgentCore agents through intelligent analysis and automated code transformation.
+
+**🔄 Agent Code Transformation:**
+* **Analyze existing agents** - Detect frameworks, patterns, and migration strategies
+* **Transform to AgentCore** - Automated code conversion with best practices
+* **Framework agnostic** - Works with any Python-based agent framework
+
+**🚀 Production Deployment:**
+* **One-click deployment** with OAuth authentication and memory capabilities
+* **Smart invocation** with automatic fallback between authentication methods
+* **Status monitoring** and runtime health checks
+
+**🔧 Infrastructure Management:**
+* **Gateway creation** for MCP-compatible integrations with AWS services
+* **Credential management** with secure API key storage and runtime injection
+* **Memory integration** with semantic and summary strategies
+
+**🎯 Developer Experience:**
+* **15 specialized tools** for every stage of agent development
+* **Live tool inspection** with MCP Inspector integration
+* **Coding assistant integration** (VS Code, Amazon Q Developer, Kiro, Cline, Cursor, Windsurf, etc.)
+
+### Pre-Requisites:
+
+1. [Sign-Up for an AWS account](https://aws.amazon.com/free/?trk=78b916d7-7c94-4cab-98d9-0ce5e648dd5f&sc_channel=ps&ef_id=Cj0KCQjwxJvBBhDuARIsAGUgNfjOZq8r2bH2OfcYfYTht5v5I1Bn0lBKiI2Ii71A8Gk39ZU5cwMLPkcaAo_CEALw_wcB:G:s&s_kwcid=AL!4422!3!432339156162!e!!g!!aws%20sign%20up!9572385111!102212379327&gad_campaignid=9572385111&gbraid=0AAAAADjHtp99c5A9DUyUaUQVhVEoi8of3&gclid=Cj0KCQjwxJvBBhDuARIsAGUgNfjOZq8r2bH2OfcYfYTht5v5I1Bn0lBKiI2Ii71A8Gk39ZU5cwMLPkcaAo_CEALw_wcB)
+2. [Set up Amazon Bedrock Agent Core](https://docs.aws.amazon.com/bedrock/latest/userguide/agents.html) access in your AWS account
+3. Install `uv` from [Astral](https://docs.astral.sh/uv/getting-started/installation/) or the [GitHub README](https://github.com/astral-sh/uv#installation)
+4. Install Python using `uv python install 3.10`
```

Comment:
Since using `uv` there should be no need to install `python`

Reply:
removed.